kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -147,6 +157,9 @@ private[kyuubi] class EngineRef(
    *   /`serverSpace_version_USER_engineType`/`user`[/`subdomain`]
    * For `GROUP` share level:
    *   /`serverSpace_version_GROUP_engineType`/`primary group name`[/`subdomain`]
+   * For `SERVER_LOCAL` share level:
+   *   /`serverSpace_version_SERVER_LOCAL_engineType`
+   *   /`kyuubi server user`/`hostAddress`[/`subdomain`]
```

Comment:
This seems to have an extra layer than the original, @pan3793 @yaooqinn is this allowed? 

Or do we add server host to subdomain? like:

```
/`serverSpace_version_SERVER_LOCAL_engineType`/`kyuubi server user`/`hostAddress`[_`subdomain`]
```

Reply:
I think it make sense to add server host to subdomain, I will update it